src/platform_impl/web/monitor.rs

```diff
@@ -128,6 +128,10 @@ impl MonitorHandleProvider for MonitorHandle {
         self.id.unwrap_or_default()
     }
 
+    fn physical_size(&self) -> Option<PhysicalSize<u32>> {
+        None
```

Comment:
Could you leave a comment as to why this is left unimplemented?

Reply:
Left a comment.